src/tranqu/transpiler/transpiler_manager.py

```diff
@@ -24,6 +28,40 @@ class TranspilerManager:
 
     def __init__(self) -> None:
         self._transpilers: dict[str, Any] = {}
+        self._default_transpiler_lib: str | None = None
+
+    def register_default_transpiler_lib(
+        self,
+        default_transpiler_lib: str,
+    ) -> None:
+        """Register the default transpiler library.
+
+        Args:
+            default_transpiler_lib (str): The name of the default transpiler library
+                to register.
+
+        Raises:
+            DefaultTranspilerLibAlreadyRegisteredError: If a transpiler
+                with the same name is already registered.
+
+        """
+        if self._default_transpiler_lib is not None:
+            msg = (
+                f"Default transpiler_lib '{self._default_transpiler_lib}' "
+                "is already registered."
+            )
+            raise DefaultTranspilerLibAlreadyRegisteredError(msg)
+
+        self._default_transpiler_lib = default_transpiler_lib
+
+    def fetch_default_transpiler_lib(self) -> str | None:
```

Comment:
I think this function should be named `get_default_transpiler_lib` instead of `fetch_default_transpiler_lib`.
In the Tranqu codebase, we distinguish between the `get_` and `fetch_` prefixes as follows:
- get: returns None if not found
- fetch: raises an exception if not found